src/form-elements/FormElementGoogleAddress.tsx

```diff
@@ -62,27 +62,35 @@ function FormElementGoogleAddress({
               input,
             },
             (predictions, status) => {
-              if (
-                status === google.maps.places.PlacesServiceStatus.ZERO_RESULTS
-              ) {
-                resolve([])
-                return
-              }
-              if (status !== google.maps.places.PlacesServiceStatus.OK) {
-                reject('Google Places service not available')
-                return
+              switch (status) {
+                case google.maps.places.PlacesServiceStatus.ZERO_RESULTS: {
+                  resolve([])
+                  break
+                }
+                case google.maps.places.PlacesServiceStatus.REQUEST_DENIED: {
+                  reject(
+                    'Google Maps API key has not been configured correctly',
```

Comment:
bad practice to reject/throw anything other than an `Error`.

Reply:
yah fair enough, fixing